api/tests/routes/backoffice/non_payment_notices_test.py

```diff
@@ -0,0 +1,284 @@
+        assert rows[2]["Date de création"] == closed_notice.dateCreated.strftime("%d/%m/%Y")
+        assert rows[2]["Date de réception"] == closed_notice.dateReceived.strftime("%d/%m/%Y")
+        assert rows[2]["État"] == "Terminé"
+        assert rows[2]["Type d'avis"] == "Huissier de justice"
+        assert rows[2]["Référence"] == "ABC123"
+        assert rows[2]["Nom de l'émetteur"] == "Guy Ssier de Justice"
+        assert rows[2]["Mail de l'émetteur"] == "plus.dargent@example.com"
+        assert rows[2]["Montant"] == "199,99 €"
+        assert rows[2]["Entité juridique"] == closed_notice.offerer.name
+        assert rows[2]["Partenaire culturel"] == closed_notice.venue.name
+        assert rows[2]["Motif"] == "Déjà payé"
+        assert rows[2]["N° de virement"] == "VIR123"
+
+
+class GetCreateNonPaymentNoticeFormTest(GetEndpointHelper):
+    endpoint = "backoffice_web.non_payment_notices.get_create_non_payment_notice_form"
+    needed_permission = perm_models.Permissions.MANAGE_NON_PAYMENT_NOTICES
+
+    def test_get_create_form_test(self, legit_user, authenticated_client):
+        form_url = url_for(self.endpoint)
+
+        with assert_num_queries(2):  # session + current user
+            response = authenticated_client.get(form_url)
+            assert response.status_code == 200
+
+
+class CreateNonPaymentNoticeTest(PostEndpointHelper):
+    endpoint = "backoffice_web.non_payment_notices.create_non_payment_notice"
+    needed_permission = perm_models.Permissions.MANAGE_NON_PAYMENT_NOTICES
+
+    # avec offerer et venue
+    # avec offerer sans venue
+    # avec venue sans offerer
+
+    def test_create_non_payment_notice(self, authenticated_client):
+        response = self.post_to_endpoint(
+            authenticated_client,
+            form={
+                "date_received": datetime.date.today() - datetime.timedelta(days=1),
+                "notice_type": "BAILIFF",
+                "amount": "12,34",
+                "reference": "123ABC",
+                "emitter_name": "Ferran Largent",
+                "emitter_email": "avis.impayé@example.com",
+                "offerer": "",
+                "venue": "",
+            },
+        )
+
+        assert response.status_code == 303
+        assert (
+            html_parser.extract_alert(authenticated_client.get(response.location).data) == "L'avis d'impayé a été créé"
+        )
+
+        notice = db.session.query(offerers_models.NonPaymentNotice).one()
+        assert notice.dateReceived == datetime.date.today() - datetime.timedelta(days=1)
+        assert notice.status == offerers_models.NoticeStatus.CREATED
+        assert notice.noticeType == offerers_models.NoticeType.BAILIFF
+        assert notice.reference == "123ABC"
+        assert notice.emitterName == "Ferran Largent"
+        assert notice.emitterEmail == "avis.impayé@example.com"
+        assert notice.amount == Decimal("12.34")
+        assert notice.offerer == None
```

Comment:
Par homogénéité du code, ici et pour tous les autres plus bas : 
```suggestion
        assert notice.offerer is None
```